backends/gdx-backend-lwjgl3/build.gradle

```diff
@@ -19,6 +19,7 @@ targetCompatibility = 1.8
 
 dependencies {
     compile libraries.lwjgl3
+    compile ('com.github.tisoft:lwjgl3-awt:727caee400')
```

Comment:
Since we don't need the Vulkan support, we could exclude it

Reply:
I have added a PR to lwjgl3-awt to mark both lwjgl-opengl and lwjgl-vulkan as optional in the pom. This would fix the dependency-bloat issue.

https://github.com/LWJGLX/lwjgl3-awt/pull/27

Maybe you want to support that PR. 😄

Reply:
Haha, already merged 👯

Reply:
And a release was cut, so I will move to the released version, as soon as it appears in the repos

Reply:
Nice 👍

Reply:
release cutting seems to have failed, but we have a snapshot version in OSS sonatype now.